src/caret_analyze/plot/bokeh/histogram.py

```diff
@@ -0,0 +1,51 @@
+from bokeh.plotting import figure, show
+
+from caret_analyze.exceptions import UnsupportedTypeError
+
+from caret_analyze.record import ResponseTime
+
+from .histogram_interface import HistPlot
+
+
+class ResponseTimePlot(HistPlot):
+
+    def __init__(
+        self,
+        target,
+        case='default'
+    ):
+        if case not in ['default', 'best', 'worst']:
+            raise UnsupportedTypeError(
+                f'Unsupported "case". case = {case}.'
+                'supported "case": [default/best/worst]'
+            )
+        super().__init__(target)
+        self._case = case
+
+    def _show_core(self):
+        for i, path in enumerate(self._target, 1):
+            try:
+                records = path.to_records()
+                response = ResponseTime(records)
+            except IndexError:
+                pass
```

Comment:
Did any index error occur when you were developing?
I'm concerned old response variable is used to plot because this line ignores excepton and proceed to plot.